CHANGELOG.md

```diff
@@ -4,10 +4,20 @@ The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and p
 
 ## 2.7.0
 
+### New Features
+
+* Added support for Delta Sync Feature
+    Delta Sync allows you to perform automatic synchronization with an AWS AppSync GraphQL server. The client will perform reconnection, exponential backoff, and retries when network errors take place for simplified data replication to devices. For more details, please refer [documentation.](https://aws-amplify.github.io/docs/ios/api)
+
+### Bug Fixes
+
+* Fix issue where if a time out error occurs, the callback would not be given back to developer. [See issue #91](https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/91)
```

Comment:
Change to "Fixed issue where if a timeout error occurred due to lack of network availability, the callback would not be given back to the developer"